Makefile

```diff
@@ -64,6 +64,6 @@ ifndef CONTINUOUS_INTEGRATION
 	$(PYTHON) -m flake8 . --count --exclude=./.*,scripts/20*,vendor*,node_modules/* --exit-zero --max-complexity=10 --max-line-length=$(GITHUB_EDITOR_WIDTH) --statistics
 endif
 
-test:
+test: i18n
```

Comment:
How much longer does this take to complete? It might be ok, but I'd worry that it'd take a lot longer to perform processing that isn't strictly testing. Also, won't it possibly make changes to commitable translation files, which is not what a dev would expect when running tests?

Reply:
The timing will vary with the number of languages, but the current 5 languages take 641 msec to compile on my (few years old) laptop.

The output of the message catalog compilation is a set of derived artefacts which are not committed to git (the .gitignore includes *.mo).

Reply:
Yeah << 1s for me as well. I put it first cause it ran _significantly_ faster than the other ones :P

Reply:
Ok, that's not long at all (I thought it'd be considerably longer).

There's still something about adding a non-test task as a test in the makefile that I'm not fully comfortable with, but I accept the functionality serves as an ok test. ... The alternative would be to wrap the whole i18n task in  pytest, but that'd be more work for little gain. It's probably fine like this.